player/Player.tscn

```diff
@@ -39,6 +39,20 @@ tracks/0/keys = {
 "update": 1,
 "values": [ 0, 4, 8, 12 ]
 }
+tracks/1/type = "method"
```

Comment:
i don't think we should rely on the animation to keep track of the direction. i think it's better if we do it in the code as it avoids the signal and it's more visible. we can have something like the following in the `Player`'s `_PhysicsProcess`:

```csharp
var x = movement.x > 0 ? 1 : (movement.x < 0 ? -1 : 0);
var y = movement.y > 0 ? 1 : (movement.y < 0 ? -1 : 0);
GD.Print("direction=" + new Vector2(x, y));
```

what do you think?

Reply:
[8002e72](https://github.com/fourlastor-jams/game-off-2022/pull/13/commits/8002e72bdeaa2200277be3f3f6b8266eeb09cd6c) makes sense, I used `Mathf.Sign` for the direction (it should be 1/-1 all the time as the character always faces a direction)